inc/admin/laf/namespace.php

```diff
@@ -1161,6 +1161,28 @@ function ( $path ) use ( $assets ) {
 	wp_register_script( 'alpinejs', $assets->getPath( 'scripts/alpine.min.js' ), [], false, true );
 	wp_enqueue_script( 'alpinejs' );
 
+	if ( isset( $_REQUEST['page'] ) && $_REQUEST['page'] === 'pb_cloner' && \Pressbooks\Utility\are_algolia_search_env_var_presents() ) {
```

Comment:
I don't think worth to create tests for this portion of the code. What's inside are all wp core methods already tested in the central core.